src/acat/azext_acat/aaz/latest/app_compliance_automation/__cmd_group.py

```diff
@@ -0,0 +1,23 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command_group(
+    "acat",

```

Comment:
the folder name should be `acat` not `app_compliance_automation`

Reply:
It seems you add customization code directly in the generated aaz folder. It's not recommended. I think for you most of your case you only need to implement a custom client which is used to add `x-ms-aad-user-token` in your header.